python-spec/src/somacore/data.py

```diff
@@ -454,15 +461,6 @@ class SparseRead:
 
     __slots__ = ()
 
-    def coos(self) -> ReadIter[pa.SparseCOOTensor]:
-        raise NotImplementedError()
-
-    def cscs(self) -> ReadIter[pa.SparseCSCMatrix]:
-        raise NotImplementedError()
-
-    def csrs(self) -> ReadIter[pa.SparseCSRMatrix]:
-        raise NotImplementedError()
-
```

Comment:
no longer part of the spec? out-of-scope for the issue of this PR?

Reply:
oops i need to add a mod to the spec markdown file

discussion at
https://github.com/single-cell-data/TileDB-SOMA/issues/931#issuecomment-1433064704
was these simply cease to be

Reply:
https://github.com/chanzuckerberg/cell-census/blob/9a441ac232d8b5f941823bab35994702a616fb32/api/python/cell_census/src/cell_census/presence_matrix.py#L41

Reply:
and they _have_ to cease to be in a mutable-X world

Reply:
Not sure I undertand why, but I accept it. 😄

Reply:
> Issue comment says these are not used, but
I was just repeating @bkmartinjr 's assertion -- i didn't audit the cell-census code

IMO (IIUC) the conversion to CSR would be a few-lines change in the cell-census code ...

Reply:
> Not sure I undertand why

@atolopko-czi details at
https://github.com/single-cell-data/TileDB-SOMA/issues/931
-- in particular
https://github.com/single-cell-data/TileDB-SOMA/issues/931#issuecomment-1433064704